modules/datastore/src/Storage/DatabaseTable.php

```diff
@@ -188,7 +198,34 @@ protected function buildTableSchema($tableName, $fieldsInfo) {
       ];
       $schema['fields'][$name] = array_filter($schema['fields'][$name]);
     }
-    return $schema ?? ['fields' => []];
+    // Add index information to schema if available.
+    $this->addIndexInfo($schema);
+
+    return $schema;
+  }
+
+  /**
+   * Add index information to table schema.
+   *
+   * @param array $schema
+   *   Drupal Schema API array.
+   */
+  protected function addIndexInfo(array &$schema) {
```

Comment:
just adding a return type
```suggestion
  protected function addIndexInfo(array &$schema): void {
```